sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -36,6 +36,7 @@ export enum AppRoutes {
   LocationProgramsOfferingsCreate = "location-offerings/location/:locationId/education-program/:programId/create",
   LocationUsers = "location-users/",
   ActiveApplicationsSummary = "active-applications/location/:locationId/summary",
+  RequestAChange = "request-change/location/:locationId/summary",
```

Comment:
RequestChange?

Reply:
we already had a RequestChange, 
![image](https://github.com/bcgov/SIMS/assets/77353155/fab4aff0-8545-4aad-b768-9ed86638caac). I think I will give a different name to avoid confusion, even though there is an institution comment